packages/@aws-cdk/cdk/lib/util/uniqueid.ts

```diff
@@ -0,0 +1,85 @@
+// tslint:disable-next-line:no-var-requires
+const md5 = require('./md5');
```

Comment:
Please switch to use `crypto` instead. And although I know that's not a security-sensitive context, it mains me a little every time I see `MD5`.

Reply:
Done